operator/pkg/central/reconciler/reconciler.go

```diff
@@ -60,7 +61,9 @@ func RegisterNewReconciler(mgr ctrl.Manager, selector string) error {
 		mgr, platform.CentralGVK, image.CentralServicesChartPrefix,
 		translation.WithEnrichment(
 			centralTranslation.New(mgr.GetClient()),
-			proxy.NewProxyEnvVarsInjector(proxyEnv, mgr.GetLogger())),
+			proxy.NewProxyEnvVarsInjector(proxyEnv, mgr.GetLogger()),
+			legacy.NewImagePullSecretReferenceInjector(mgr.GetClient(), "imagePullSecrets",
+				"stackrox", "stackrox-scanner", "stackrox-scanner-v4")),
```

Comment:
I have mixed feelings about this. The whole point for supporting these various secrets is a mystery to me. However if there is a v4-specific service account and pull secret, then I think it's only appropriate to treat them in a manner consistent with how scanner(v2 ?) counterparts are. I guess to avoid introducing yet another special case, even if its utility is minimal.